spec/common/related.html

```diff
@@ -1,9 +1,7 @@
 <h4>Set of Documents</h4>
-<p>This document is one of ten RDF 1.2 and twelve SPARQL 1.2 documents produced by the <a href="https://www.w3.org/groups/wg/rdf-star">RDF-star Working Group</a>.</p>
-<details>
-<summary>List of documents</summary>
+<p>This document is one of ten RDF 1.2 and twelve SPARQL 1.2 Recommendations produced by the <a href="https://www.w3.org/groups/wg/rdf-star">RDF-star Working Group</a>:</p>
 
-<p>RDF 1.2 Documents:</p>
+<p>RDF 1.2 Recommendations:</p>
 <ol>
   <li data-transform="noSelfCite">[[[RDF12-NEW]]]</li>
```

Comment:
nit: this is not a recommendation (same for the primer and sparql-new)

Reply:
My mistake. I updated the `spec/common/` area from an out-of-date rdf-common.

I reset the PR to be an update from current `rdf-common`.